templates/supdfgen/innkallingKontrollsamtale.hbs

```diff
@@ -0,0 +1,39 @@
+{{#> supdfgen/partials/base}}
+    <div class="mainContent">
+        <h3>Supplerende stønad ufør flyktning – innkalling til samtale</h3>
+        {{> supdfgen/partials/fnr personalia }}
+
+        <p>
+            I forbindelse med saken din ønsker vi en samtale med deg på NAV-kontoret. Koronasituasjonen er bedret og samfunnet har begynt å åpne opp igjen.
+            Koronaunntakene avvikles. Du må derfor møte personlig på NAV-kontoret.
+        </p>
+
+        <p>
+            Du må møte snarest og senest innen 3 uker fra du får dette brevet.
+        </p>
+
+        <p>
+            Du ringer NAV Kontaktsenter på 55 55 33 33 for å avtale tidspunkt for et møte. Hvis du trenger tolk, må du ta kontakt snarest,
+            slik at NAV-kontoret kan innkalle tolk til samtalen. Hvis NAV-kontoret tidligere har sagt at du kan møte opp uten timeavtale, kan du forholde deg til det.
+        </p>
+
+        <p>
+            Dersom du ikke møter eller tar kontakt med oss, vil din supplerende stønad bli stanset, jf. paragrafene 18 og 21 i lov om supplerende stønad til personer med kort botid i Norge.
+        </p>
+
+        <p>
+            Du får ikke ny melding dersom stønaden din stanses.
+        </p>
+
+        Du må ha med følgende dokumentasjon:
+        <ul>
+            <li>Alle gyldige pass</li>
+            <li>Dokumentasjon på ut- og hjemreise dersom du har vært i utlandet i løpet av de fire siste månedene.</li>
```

Comment:
Jeg antar at vi ikke kan garantere at det er 4 måneder siden brukeren hadde kontakt med nav. Burde vi hatt en litt mer referansebasert tilnærming her? F.eks:
```suggestion
            <li>Dokumentasjon på ut- og hjemreise dersom du har vært i utlandet siden søknadstidspunktet eller forrige kontrollsamtale.</li>
```

Reply:
Vet ikke om jeg orker å pirke på disse tekstene, må involvere Ellen isåfall.